reference/artifacts-with-artifactsrewrite/in-cloud-foundry/index.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: single
+title: "Artifacts In Cloud Foundry"
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+In Cloud Foundry, artifacts can used for both an app's manifest and for the deployment archive (e.g. a JAR file for a Java app). These both can be used in a Deploy stage.
+
+## Artifacts Referencing App Manifest
```

Comment:
@fifthposition 
Can you change headings, throughout all of this, to use sentence case?
This one would be "Artifacts referencing app manifest"